src/main/java/com/gengo/client/GengoConstants.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * 
+ */
+package com.gengo.client;
+
+/**
+ * @author hiroyu-n@inter-plug.co.jp
+ *
+ */
+public final class GengoConstants {
+	protected static String BASE_URL_STANDARD = "https://api.gengo.com/v2/";
```

Comment:
@yugui If the variable accessor is `private` then GengoClient.java can't refer it.